client/jetpack-cloud/style.scss

```diff
@@ -320,6 +320,20 @@ html {
 
 .site-selector {
 	display: block;
+	.site-selector__actions {
+		padding: 16px;
+	}
+	.site-selector__add-new-site {
+		border-color: var( --color-sidebar-text );
+		border-radius: 2px;
```

Comment:
<img width="401" alt="CleanShot 2022-09-07 at 09 50 26@2x" src="https://user-images.githubusercontent.com/1500769/188747260-58ef9957-d8a6-4361-a61f-f88db1f5a5af.png">

Looks like Jetpack Cloud has it's own border radius value, so this rule ends up just getting ignored.